apps/src/blockly/addons/cdoConstantsProvider.ts

```diff
@@ -1,8 +1,15 @@
-import GoogleBlockly, {Connection} from 'blockly/core';
-import {PuzzleTab} from 'blockly/core/renderers/common/constants';
+import * as GoogleBlockly from 'blockly/core';
 
 import {customConnectionBlockTypes} from './cdoConstants';
 
+interface PuzzleTab {
```

Comment:
Is the PuzzleTab type not be exported from blockly/core?

Reply:
Oh, I meant to leave a comment about these! Correct, they don't seem to be imported. I think this is something Maribeth asked us to notify her about so that they could start exporting what we need.

Reply:
Is this the same thing? https://github.com/google/blockly/blob/60da7d87625080ee44093c45a56689456883d6f6/core/renderers/common/constants.ts#L52

Reply:
Yes! For each of these example, they are being exported from that file and used (sparingly) across Blockly core, but I can't find a way to reference them off of the imported `GoogleBlockly` object. Offline, @molly-moen confirmed this. We discussed adding todos on each of these and filing a ticket with Blockly to add these to their exports.

Reply:
That sounds like a good plan to me!